### DISCUSSION TITLE
Support for RedHat Enterprise Linux 9

### DISCUSSION THREAD

**Discussion by dreiucker** [rabbitmq/rabbitmq-server#8554] [Other] [answered]
Hello, 

I'm looking at https://www.rabbitmq.com/install-rpm.html#supported-distributions I couldn't find that RedHat Enterprise Linux 9 is mentioned. Is there any plan to support RedHat Enterprise Linux 9.x in the near future? 

Thank you for your answer.

**Comment by michaelklishin** [comment]
There is nothing RHEL 9-specific for us to support.

The CentOS Stream 8/RHEL 8 package works perfectly fine on CentOS Stream 9 and RHEL 9.

Our [zero dependency Erlang RPM](https://github.com/rabbitmq/erlang-rpm/) has had CentOS Stream 9, Fedora 38 and Amazon Linux 2023 builds for a couple of months now.

**Reply by dreiucker** [reply]
I'm only allowed to use RHEL 9 if it is officially supported and some products need major adjustments to go to RHEL9, and therefore I wanted to clarify here. 

Looking at the documentation https://www.rabbitmq.com/install-rpm.html#supported-distributions it looks like CentOS Stream 9.x and CentOS Stream 8x, while only RedHat Enterprise Linux 8.x is listed. So it can be interpreted both ways.

But looking at https://github.com/rabbitmq/erlang-rpm/#supported-rpm-based-distributions and your answer from above it seems RedHat Enterprise Linux 9.x is officially supported.

Thanks for the fast answer.

**Reply by michaelklishin** [reply, marked as answer]
If "officially supported" means "listed in the RPM installation guide" then it is supported as of last night :)

**Reply by michaelklishin** [reply]
Not sure why https://github.com/rabbitmq/rabbitmq-website/commit/322aa6edacbbb0f15c4a184aa5b44e0b00de18d6 hasn't been deployed yet but it will be soon enough

**Reply by michaelklishin** [reply]
There was a rendering error in a different guide, I'm on it.
